octobot_evaluators/api/evaluators.py

```diff
@@ -45,30 +45,60 @@ async def create_evaluators(evaluator_parent_class,
                             matrix_id: str,
                             exchange_name: str,
                             bot_id: str,
-                            symbols_by_crypto_currencies: dict = None,
+                            symbols_by_crypto_currency_names: dict = None,
                             symbols: list = None,
                             time_frames: list = None,
+                            real_time_time_frames: list = None,
                             relevant_evaluators=CONFIG_WILDCARD) -> list:
-    return [
-        await create_evaluator(evaluator_class,
-                               tentacles_setup_config,
-                               matrix_id=matrix_id,
-                               exchange_name=exchange_name,
-                               bot_id=bot_id,
-                               cryptocurrency=cryptocurrency,
-                               symbol=symbol,
-                               time_frame=time_frame,
-                               relevant_evaluators=relevant_evaluators
-                               )
-        for evaluator_class in create_advanced_types_list(evaluator_parent_class, config)
-        for cryptocurrency in _get_cryptocurrencies_to_create(evaluator_class, symbols_by_crypto_currencies.keys())
-        for symbol in _get_symbols_to_create(evaluator_class, symbols_by_crypto_currencies, cryptocurrency, symbols)
-        for time_frame in _get_time_frames_to_create(evaluator_class, time_frames)
-    ]
-
-
-def _get_cryptocurrencies_to_create(evaluator_class, cryptocurrencies):  # TODO replace with python 3.8 by :=
-    return cryptocurrencies if cryptocurrencies and not evaluator_class.get_is_cryptocurrencies_wildcard() else [None]
+    crypto_currency_name_by_crypto_currencies = {}
```

Comment:
Is it possible to create a test that ensures that we create all the expected evaluator instances ?